jsapp/js/components/processing/transxDisplay.module.scss

```diff
@@ -3,7 +3,9 @@
 @use './singleProcessingVariables';
 
 .root {
-  margin-top: singleProcessingVariables.$s-single-processing-tabs;
+  // TODO: Use the $s-single-processing-tabs variable here somehow.
+  // Right now 60px is too big if there are more than 1 translations.
+  margin-top: sizes.$x32;
```

Comment:
After you introduce `gap` in `.displays` (in `sidebar.module.scss`), this TODO would be resolved, as margin will no longer be needed here

Reply:
Todo for later